unity-environment/Assets/ML-Agents/Examples/3DBall/Scripts/Ball3DDecision.cs

```diff
@@ -4,30 +4,21 @@
 
 public class Ball3DDecision : MonoBehaviour, Decision
 {
+	public float rotationSpeed = 2f;
     public float[] Decide(List<float> state, List<Texture2D> observation, float reward, bool done, List<float> memory)
     {
         if (gameObject.GetComponent<Brain>().brainParameters.vectorActionSpaceType == StateType.continuous)
         {
-            List<float> ret = new List<float>();
-            if (state[2] < 0 || state[5] < 0)
-            {
-                ret.Add(state[5]);
-            }
-            else
-            {
-                ret.Add(state[5]);
-            }
-            if (state[3] < 0 || state[7] < 0)
-            {
-                ret.Add(-state[7]);
-            }
-            else
-            {
-                ret.Add(-state[7]);
-            }
-            return ret.ToArray();
-
+            List<float> act = new List<float>();
+            //state[5] is the velocity of the ball in the x orientation. We use this number to control the Platform's z axis rotation speed, 
```

Comment:
- Limit is 80 characters per line
- add a space after "//"
- add an empty line to separate code blocks for improved readability

Reply:
// Limit is 80 characters per line
Fixed

// add a space after "//"
Fixed this and all the other places in the code base. 

// add an empty line to separate code blocks for improved readability
What is considered a "separate code block" in this case? I already have empty line before and after each method.